content/ri/installing/configurations.md

```diff
@@ -31,3 +31,5 @@ The following environment variables can be set to configure RedisInsight:
 - `REDISINSIGHT_HOME_DIR` - By default, RedisInsight stores application data (such as local database, log files and snapshot files) in  `~/.redisinsight` in your home directory on desktop and in the `/db` directory in the Docker container. To change this directory, set the `REDISINSIGHT_HOME_DIR` environment variable.
 
 - `LOG_DIR` - By default, RedisInsight stores log files in your home directory (`REDISINSIGHT_HOME_DIR`). To have RedisInsight store the log files in a different directory, set the `LOG_DIR` environment variable.
+
```

Comment:
@slemeur This is ambiguous. We should be clear about the default log level. Is the default log level "WARNING"? If so, we should say that explicitly.

Reply:
+1